app/views/layouts/application.html.erb

```diff
@@ -42,6 +42,14 @@
             <% if current_user %>
               <li><%= link_to "My Trainings", user_trainings_path(current_user) %></li>
             <% end %>
+            <li class="dropdown">
+            <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Places <span class="caret"></span></a>
+            <ul class="dropdown-menu">
+              <li><%= link_to "Places", places_path %></li>
+                <% if current_user %>
+              <li><%= link_to "New Place", new_place_path %></li>
```

Comment:
if it's inside the block, it should be indented one level deeper

Reply:
OK